release/models/platform/openconfig-platform.yang

```diff
@@ -1142,9 +1180,59 @@ module openconfig-platform {
+             The state/last-poweroff-reason/trigger should show as USER_INITIATED. Also the NOS can update the 
+             leaf /state/last-poweroff-reason/details. The leaf state/last-poweroff-time should record the time 
+             when the card was powered-off. Following paths:
+
+             /components/component[name=controller0]/state/redundant-role, PRIMARY
+             /components/component[name=controller1]/state/redundant-role, SECONDARY
+             /components/component[name=controller0]/controller-card/config/power-admin-state, POWER_ENABLED
+             /components/component[name=controller1]/controller-card/state/power-admin-state, POWER_DISABLED
+             /components/component[name=controller1]/controller-card/state/is-powered-off, TRUE
+             /components/component[name=controller1]/state/last-poweroff-reason/trigger, USER_INITIATED
+             /components/component[name=controller1]/state/last-poweroff-reason/details, "User initiated Shutdown"
+             /components/component[name=controller0]/state/last-poweroff-time, 1706590714681765937
+
+             Scenario#2: If controller0 is PRIMARY and controller1 is SECONDARY and if controller0 is set to 
+             config/power-admin-state = POWER_DISABLED by an operator, then controller0 will stay powered-on until the next reboot. 
+             The leaf state/power-admin-state should show as POWER_DIABLED. The boolean flag however, state/is-powered-off 
+             should show as FALSE. The state/last-poweroff-reason/trigger should show as USER_INITIATED. Also the NOS can 
+             update the leaf /state/last-poweroff-reason/details. The leaf state/last-poweroff-time should record the time 
+             when the card powers-off post the next reboot. Following paths
+
+             /components/component[name=controller0]/state/redundant-role, PRIMARY
+             /components/component[name=controller1]/state/redundant-role, SECONDARY
+             /components/component[name=controller0]/controller-card/config/power-admin-state, POWER_DISABLED
+             /components/component[name=controller1]/controller-card/state/power-admin-state, POWER_ENABLED
+             /components/component[name=controller0]/controller-card/state/is-powered-off, FALSE
+             /components/component[name=controller0]/state/last-poweroff-reason/trigger, USER_INITIATED
+             /components/component[name=controller0]/state/last-poweroff-reason/details, "User initiated Shutdown"
+             /components/component[name=controller0]/state/last-poweroff-time, 1706590714681765937
+
+          2. A controller-card which is in state/redundant-role=SECONDARY and is config/power-admin-state=POWER_DISABLED should 
+            remain powered off, even after a reboot. It is possible that after a reboot, both the controller cards are powered-on. 
+            However as soon as the configuration is loaded the NOS should power-off the subject controller-card.
```

Comment:
Suggesting we keep with 'system' as this is regularly used in openconfig models.  
```suggestion
            However as soon as the configuration is loaded the system should power-off the subject controller-card.
```

Reply:
ack!